x/bet/keeper/place_bet.go

```diff
@@ -52,8 +52,13 @@ func (k Keeper) PlaceBet(ctx sdk.Context, bet *types.Bet) error {
 
 	bet.CreatedAt = ctx.BlockTime().Unix()
 
+	stats := k.GetBetStats(ctx)
+	stats.Count += 1
+
 	// store bet in the module state
-	k.SetBet(ctx, *bet)
+	k.SetBet(ctx, *bet, stats.Count)
+	k.SetBetStats(ctx, stats)
```

Comment:
Should we move getting stats and updating stats inside set bet itself, and not expose the SetBetStats function?

Reply:
It is not possible, Because in the `InitGenesis` We have the latest `betstats` that contains the last generated ID before export.